### PR TITLE
Merged NEWS into NEWS.md

### PR DESCRIPTION
**Pull request by JoshuaSturm**
Merged old NEWS into NEWS.md, and updated reference on homepage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ packageVersion("openxlsx")
...
-[&#x0040;aavanesy](https://github.com/aavanesy), [&#x0040;ale275](https://github.com/ale275), [&#x0040;alexb523](https://github.com/alexb523), [&#x0040;david-f1976](https://github.com/david-f1976), [&#x0040;davidgohel](https://github.com/davidgohel), [&#x0040;dovrosenberg](https://github.com/dovrosenberg), [&#x0040;JoshuaSturm](https://github.com/JoshuaSturm), [&#x0040;SHAESEN2](https://github.com/SHAESEN2), [&#x0040;soliac](https://github.com/soliac), [&#x0040;theclue](https://github.com/theclue), and [&#x0040;ycphs](https://github.com/ycphs)
+[&#x0040;awalker89](https://github.com/awalker89), [&#x0040;aavanesy](https://github.com/aavanesy), [&#x0040;ale275](https://github.com/ale275), [&#x0040;alexb523](https://github.com/alexb523), [&#x0040;david-f1976](https://github.com/david-f1976), [&#x0040;davidgohel](https://github.com/davidgohel), [&#x0040;dovrosenberg](https://github.com/dovrosenberg), [&#x0040;JoshuaSturm](https://github.com/JoshuaSturm), [&#x0040;SHAESEN2](https://github.com/SHAESEN2), [&#x0040;soliac](https://github.com/soliac), [&#x0040;theclue](https://github.com/theclue), and [&#x0040;ycphs](https://github.com/ycphs)
```
Comment (ycphs) [inline README.md:50]: What is the change here?

File: NEWS.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 * include tests for cloneWorksheet
 
+* Fixed NEWS hyperlink
+
 # openxlsx 4.1.4
 
 * Use `zip::zipr()` instead of zip::zip().
```
Comment (ycphs) [inline NEWS.md]: * Use `zip::zipr()` instead of `zip::zip()`.

Could you also correct this line?

File: NEWS.md [hunk 3]
```diff
@@ -20,4 +22,341 @@
...
+
+* date detection basic_string error
+
+* multiple spaces in table column names were not being maintained thus corrupting the xlsx file.
+
+* openXL fail silently on relative paths
+
+* `headerStyle` failed when writing a list of length 1 using write.xlsx
+
+* `detectDate` for read.xlsx issues
```
Comment (ycphs) [inline NEWS.md]: * `detectDate` for `read.xlsx` issues

Could you also correct this line?

**Review by ycphs — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 3 above)
Comment (ycphs) [reply NEWS.md]: If you use 2 white space between the asterix and the text the Codacy/PR Quality Review check will be successful.

File: README.md (hunk 1 above)
Comment (JoshuaSturm) [reply README.md:50]: I added awalker to the list.

Comment (ycphs) [reply README.md:50]: Thank you

**Comment by ycphs** [comment]
Thank you very much

**Pull request merged**